MUXControlsInnerLoop.sln

```diff
@@ -991,6 +987,7 @@ Global
 		{1A5321F3-B837-4EB6-9547-37CC70088EA9} = {C5EFA126-F329-4F8B-8DC9-E4182F4976D0}
 		{E95C2CA1-FF23-47CC-A896-CC5175B37890} = {67599AD5-51EC-44CB-85CE-B60CD8CBA270}
 		{9D23C997-1F46-444A-8C07-4A4BFF7E4E63} = {E95C2CA1-FF23-47CC-A896-CC5175B37890}
+		{80AF98CA-BC1D-4011-8460-5671799EC419} = {E95C2CA1-FF23-47CC-A896-CC5175B37890}
```

Comment:
This is the actually relevant line for this PR here, playing the NumberBox's APITests project correctly in the project structure now.

Reply:
Strangely, #3146 also introduced that line. We lost that loc with #2977 though.